backend/migrations/versions/11b716ddb036_backfill_get_org_permissions.py

```diff
@@ -0,0 +1,50 @@
+"""backfill_get_org_permissions
+
+Revision ID: 11b716ddb036
+Revises: 458572580709
+Create Date: 2024-05-22 17:06:34.441667
+
+"""
+
+from alembic import op
+from sqlalchemy import orm
+from dataall.core.organizations.db.organization_models import Organization
+from dataall.core.environment.db.environment_models import EnvironmentGroup, Environment
+from dataall.core.permissions.services.resource_policy_service import ResourcePolicyService
+from dataall.core.permissions.services.organization_permissions import GET_ORGANIZATION
+
+
+# revision identifiers, used by Alembic.
+revision = '11b716ddb036'
+down_revision = '458572580709'
```

Comment:
I see the migration script is failing because of multiple heads in alembic ... I believe you will need to merge the latest from `main` to this branch and then update the `down_revision` here to be pointing from the latest revision currently in main